src/bmi_DAforcing.py

```diff
@@ -59,6 +59,85 @@ def __init__(self):
         'lastobs_df'
     ]
 
+    #---------------------------------------------
+    # BMI variable output namesnames (CSDMS standard names)
+    #---------------------------------------------
+    # USGS DF
+    _BMI_output_usgs = [
+        'datesSecondsArray_usgs',
+        'nDates_usgs',
+        'stationArray_usgs',
+        'stationStringLengthArray_usgs',
+        'nStations_usgs',
+        'usgs_Array'
+    ]    
+    
+    # Reservoir USGS DF
+    _BMI_output_reservoir_usgs = [
+        'datesSecondsArray_reservoir_usgs',
+        'nDates_reservoir_usgs',
+        'stationArray_reservoir_usgs',
+        'stationStringLengthArray_reservoir_usgs',
+        'nStations_reservoir_usgs',
+        'reservoir_usgs_Array'
+    ]
+
+    # Reservoir USACE DF
+    _BMI_output_reservoir_usgs = [
+        'datesSecondsArray_reservoir_usace',
+        'nDates_reservoir_usace',
+        'stationArray_reservoir_usace',
+        'stationStringLengthArray_reservoir_usace',
+        'nStations_reservoir_usace',
+        'reservoir_usace_Array'
+    ]
+
+    # RFC DF
+    _BMI_output_reservoir_usgs = [
```

Comment:
Isn't this rfc so _rfc instead of _usgs?

Reply:
Fixed it,, thanks!